### ISSUE TITLE
Behaviour of meatball icon in server tab is not intuitive

### ISSUE THREAD

**Issue by mestaritonttu** [kiwiirc/kiwiirc#333]
When you hover over the meatball icon, it explodes into three different icons. This is contrary to the usual behaviour. Users expect to click on such an element before something happens. Please reconsider the behaviour.

**Comment by AllanOcelot** [comment]
Hi @mestaritonttu , how about this?
![image](https://user-images.githubusercontent.com/11334905/58093438-6cc37280-7bc6-11e9-8e38-98616a26aa35.png)

**Comment by mestaritonttu** [comment]
Now there are only two buttons behind the meatball. Do we need to keep them hidden by default? I don't think there would be a feeling of clutter, if we just displayed both. Or is there a possibility that the number will grow in the future?

If not both, would it be silly, if only the search button was displayed and then it would reveal the network-adding button upon hover?

I tried searching for examples that match this interaction, but failed. The closest was a stack of icons, such as social media buttons that expand on hover: https://codemyui.com/stacked-social-media-icons-expand-on-hover/
